AI/VCAI/AIUtility.h

```diff
@@ -35,7 +35,7 @@ extern const int GOLD_RESERVE;
 //provisional class for AI to store a reference to an owned hero object
 //checks if it's valid on access, should be used in place of const CGHeroInstance*
 
-struct HeroPtr
+struct DLL_EXPORT HeroPtr

```

Comment:
All this DLL_EXPORT will not work, because you need it to be DLL_IMPORT inside tests. You need other macro, that work same way as DLL_LINKAGE for VCMILib

Reply:
You`re lucky :P DLL_LINKAGE macro is there on purpose ;-)

Reply:
DLL_LINKAGE didn't work, since in VCAI it's set to import

Reply:
Please, read my message again "You need other macro, that work same way ..."